narwhals/_dask/group_by.py

```diff
@@ -125,48 +125,48 @@ def agg_dask(
             all_simple_aggs = False
             break
 
-    if all_simple_aggs:
```

Comment:
this is just a drive-by to reduce the indentation, if you look at changes with "ignore whitespace" you'll see what i mean